frontend/src/lib/api/getData.ts

```diff
@@ -156,6 +157,13 @@ export const getNominatedCandidates = ({
           },
           politicalExperience: attr.politicalExperience
         };
+
+        let photoURL = attr?.photo?.data?.attributes?.url;
+        if (photoURL) {
```

Comment:
I guess we're going for a quick solution :smile:, but for a more permanent solution, we might want to save also the alt text, width, height and some of the different resolution (thumbnail and large, for example) data in a `photo` property.

Reply:
Yeah sure that could be done, we didn't have any plans when I started creating this but I'll add these to the PR!